src/smc-util/message.coffee

```diff
@@ -1034,13 +1068,33 @@ message # client ← hub
 Queries directly to the database (sort of like Facebook's GraphQL)
 ###
 
-API message
+API message2
     event          : 'query'
-    id             : undefined
-    query          : required
-    changes        : undefined
-    multi_response : false
-    options        : undefined
+    desc           : "This queries directly the database (sort of Facebook's GraphQL)"
+    fields:
+        id:
+             init  : undefined
+             desc  : 'A unique UUID for the query'
+        query:
+             init  : required
+             desc  : 'The actual query'
+        changes:
+             init  : undefined
+             desc  : ''
+        multi_response:
+             init  : false
+             desc  : ''
+        options:
+             init  : undefined
+             desc  : ''
+    examples: [
```

Comment:
Maybe examples should be commented out or have a comment that these are not valid examples?